conf/machine/include/apalis-imx8.inc

```diff
@@ -0,0 +1,3 @@
+DISTRO_FEATURES:remove = " 3g alsa bluetooth wifi nfc "
+MACHINE_FEATURES:remove = " alsa bluetooth wifi "
```

Comment:
For future reference. Distro and machine features cannot be removed in image recipe. 
https://yocto.yoctoproject.narkive.com/zQ11Cl29/best-way-to-remove-distro-features

This entry under local.conf allows including additional conf data:
https://github.com/ni/meta-ateccgen2/blob/main/buildconf/local.conf#L275